src/measure_theory/integral/set_integral.lean

```diff
@@ -839,6 +839,15 @@ L.to_continuous_linear_map.integral_comp_comm' L.antilipschitz _
 
 end linear_isometry
 
+namespace continuous_linear_equiv
+
+variables [complete_space F] [normed_space ℝ F] [complete_space E] [normed_space ℝ E]
+
+lemma integral_comp_comm (L : E ≃L[𝕜] F) (φ : α → E) : ∫ a, L (φ a) ∂μ = L (∫ a, φ a ∂μ) :=
+L.to_continuous_linear_map.integral_comp_comm' L.antilipschitz _
```

Comment:
You can drop one of `complete_space` assumptions and transfer it via `L` (optional).

Reply:
The spaces have to be complete for the integrals to make sense. I could write `by exactI ...`, but it would just make the statement harder to read, so I'd rather keep it like that.